# output all states of a robot

string robot_id

nav_msgs/Odometry chassis_odom
# theta, w, dw
geometry_msgs/Point gimbal_odom

bool alive
bool movable
bool shootable
float32 punish_time

uint16 health
uint16 bullets
uint16 heat

# PROBABLY INFO ABOUT ENEMY DETECTION



